README.rst

```diff
@@ -85,6 +85,12 @@ Setup (integrate in an existing django project)
         'openwisp_users',
     ]
 
+Set ``EMAIL_PORT`` in ``settings.py`` to a port number. For example:
```

Comment:
If put here it could be confused as a mandatory setting, while this would be normally done only during development, therefore could you move this suggestion to "Testing (locally)" (which I also suggest to rename, see other comment).